src/meta/user/users.ts

```diff
@@ -73,18 +73,18 @@ export const profilePictureUri = (userId: number) => ApiEndPoint.User.profilePic
 export const validProfilePicture = (file: File) => !file || file.size <= 1000000
 
 // validation methods
-export const validName = (user: Partial<User>) => !Objects.isEmpty(user.name)
-export const validRole = (user: Partial<User>) => !Objects.isEmpty(user.roles)
+export const validName = (value: string) => !Objects.isEmpty(value)
+export const validRole = (value: string) => !Objects.isEmpty(value)
```

Comment:
are they used elsewhere ? they should take as input the user.

Reply:
reverted, that way was useful for the front-end part but we can handle it later

Reply:
I have also seen that validEmail is replicated somewhere else. that should be optimized and centralized in another pr. I've noted it.